README.md

```diff
@@ -43,6 +43,8 @@ cmake .. \
     && ninja install
 ```
 
+`CMAKE_BUILD_TYPE` options are: `Release, Debug, RelWithDebInfo, MinSizeRel`
```

Comment:
nit: add a `.` at the end of the sentence. Also, surround each with build type with its own set of ticks, e.g., `A`, `B`, `C`.